programs/Makefile

```diff
@@ -213,20 +213,21 @@ zstd-noxz : LZMALD  :=
 zstd-noxz : LZMA_MSG := - xz/lzma support is disabled
 zstd-noxz : zstd
 
-# note : the following target doesn't build
+## zstd-dll: zstd executable linked to dynamic library libzstd (must already exist)
+# note : the following target doesn't link
 #        because zstd uses non-public symbols from libzstd
 #        such as XXH64 (for benchmark),
-#        ZDICT_trainFromBuffer_unsafe_legacy for dictionary builder
-#        and ZSTD_cycleLog (likely for --patch-from)
-#        It's unclear at this stage if this is a scenario we want to support
-## zstd-dll: zstd executable linked to dynamic library libzstd (must already exist)
+#        ZDICT_trainFromBuffer_unsafe_legacy (for dictionary builder)
+#        and ZSTD_cycleLog (likely for --patch-from).
+#        It's unclear at this stage if this is a scenario that must be supported
 .PHONY: zstd-dll
 zstd-dll : LDFLAGS+= -L$(ZSTDDIR) -lzstd
 zstd-dll : ZSTDLIB_FILES =
 zstd-dll : $(ZSTD_CLI_OBJ)
 	$(CC) $(FLAGS) $^ -o $@$(EXT) $(LDFLAGS)
 
 
+## zstd-pgo: zstd executable optimized with pgo. `gcc` only.
```

Comment:
Bit of a shame -- is there a particular reason for that? The online documentation for clang does mention PGO at https://clang.llvm.org/docs/UsersManual.html#id38.

Reply:
maybe because the way `clang`'s pgo works is different enough to require a different script

Reply:
Still doesn't make sense. Let me try it on my end.

Reply:
Got it. Making a PR.